go.mod

```diff
@@ -39,6 +39,9 @@ require (
 require (
 	code.cloudfoundry.org/clock v0.0.0-20180518195852-02e53af36e6c // indirect
 	github.com/Azure/azure-sdk-for-go/sdk/internal v1.0.1 // indirect
+	github.com/Azure/azure-sdk-for-go/sdk/keyvault/azsecrets v0.10.1 // indirect
```

Comment:
Not sure if you need to run `go mod tidy`, I would have expected this to be a direct reference?